lib/types/SSITypesBuilder.ts

```diff
@@ -104,35 +118,35 @@ export class SSITypesBuilder {
   }
 
   private static mapExternalVerifiableCredentialToWrappedVc(
-    verifiableCredential: IVerifiableCredential | JwtDecodedVerifiableCredential | string
+    verifiableCredential: IVerifiableCredential | JwtDecodedVerifiableCredential | CompactJWT
```

Comment:
Could be replaced by OriginalVerifiableCredential

Reply:
changed